src/assets/img/fullscreen-exit.svg

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Are these the same buttons as in Lara?  Kiley wants to ensure we use the same buttons in the UI.

Reply:
Yes, they are.